tests/app/submitter/test_converter.py

```diff
@@ -104,6 +104,18 @@ def test_convert_answers_flushed_flag_overriden_to_true(self):
 
             self.assertTrue(answer_object['flushed'])
 
+    def test_started_at_should_be_set_in_payload(self):
+        with self._app.test_request_context():
+
+            questionnaire = {
+                'survey_id': '021',
+                'data_version': '0.0.2'
+            }
+
+            answer_object = convert_answers(self.metadata, QuestionnaireSchema(questionnaire), AnswerStore(), {})
```

Comment:
This is a good test - it doesn't load a whole survey to use those couple of fields!

It would be helpful to add another test that deletes the `started_at` field before calling convert_answers. The benefit of this would be that future refactors or changes where someone wants to make it rely on having `started_at` in the metadata would fail, leading them to see your test and realise it's like that for a reason!

Reply:
If `started_at` isn't in the metadata then it won't fall over, it just won't be included as I don't want to tank submissions that are in flight.  I'll still do a test that shows that if it's not in the metadata then it wont appear in the final submission.

Reply:
I like the consideration of "in flight" data